base/_mixins.scss

```diff
@@ -103,155 +146,351 @@
+///   // display: block; visibility: visible;
+///
 @mixin show( $display: block ) {
 	display: $display;
 	visibility: visible;
 }
 
-// Maintain ratio mixin. Great for responsive grids, or videos.
-// "Crops" the element to the size of the container with overflow: hidden
-// https://gist.github.com/brianmcallister/2932463
-// $ratio - Ratio the element needs to maintain.
-// Example: A 16:9 ratio would look like this: .element { @include maintain-ratio(16 9); }
-// @jhogue
+/// @author jhogue
+///
+/// @param {None}
+/// 
+/// @example scss - UsAGE
+///   // @include hide()
+///
+/// @example css - Output
+///   // display: none; visibility: hidden;
+///
+@mixin hide() {
+	display: none;
+	visibility: hidden;
+}
+
+
+/// maintain-ratio(), proportional-container)_ and image-cover() = 
+///
+/// Set a container to crop an element and maintain an aspect ratio
+/// Use proportional-container() and image-cover() together:
+///
+/// @link https://gist.github.com/brianmcallister/2932463
+///
+/// @example scss - Usage
+///   // .container {
+///   //   @include proportional-container(16 9);
+///   //   
+///   //   .element {
+///   //     @include image-cover();
+///   //   }
+///   // }
+///
+/// @example css - Output
+///   // .container {
+///   //   position: relative;
+///   //   overflow: hidden;
+///   //   z-index: 1;
+///   //   width: 100%;
+///   //   height: 0;
+///   //   padding-bottom: 56.25%;
+///   // }
+///   // .container .element {
+///   // 	transform: translateX(-50%);
+///   // 	top: 0;
+///   // 	left: 50%;
+///   // 	width: auto;
+///   // 	max-width: none;
+///   // 	height: 100%;
+///   // }
+///
+
+
+/// maintain-ratio() =
+/// Helper mixin that calculates a percentage padding value to "fix" the proportions of a container to an aspect ratio
+/// @author jhogue
+///
+/// @param {list} Ratio [1 1]
+///
+/// @require {function} length
+/// @require {function} percentage
+/// @require {function} nth
+///
+/// @example scss - Usage
+///  // @include maintain-ratio(16 9)
+///
+/// @example css - Output
+///  // width: 100%; height: 0; padding-bottom: 56.25%;
+///
 @mixin maintain-ratio($ratio: 1 1) {
 	@if length($ratio) < 2 or length($ratio) > 2 {
 		@warn "$ratio must be a list with two values.";
 	}
+	width: 100%;
 	height: 0;
 	padding-bottom: percentage(nth($ratio, 2) / nth($ratio, 1));
-	width: 100%;
 }
 
-// Apply as a typical container mixin
+/// proportional-container() =
+/// Sets a container to "crop" an element to a fixed aspect ratio
+/// @author jhogue
+///
+/// @param {list} Ratio [1 1]
+///
+/// @require {mixin} maintain-ratio
+///
+/// @example scss - Usage
+///  // @include proportional-container(16 9)
+///
+/// @example css - Output
+///  // position: relative; overflow: hidden; z-index: 1; height: 0; padding-bottom: 56.25%; width: 100%
+///
 @mixin proportional-container( $ratio: 1 1 ) {
-	overflow: hidden;
 	position: relative;
+	overflow: hidden;
 	z-index: 1;
 	@include maintain-ratio( $ratio );
```

Comment:
should this be at the top of the list per standards, or if it needs to be al the bottom can you add a comment to why?